### PR TITLE
content: add kubernetes blog

### PR DESCRIPTION
**Pull request by olehpratsko**

### CODE REVIEW

**Review by pditommaso — COMMENTED** [review]
Please keep it on hold. There's a post on the same topic on Seqera blog out today and some content needs to be reviewed

File: content/blog/2023/nextflow-kubernetes-enhancements.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+type=post
+description=It seems that Kubernetes is everywhere we look! Nextflow users increasingly see K8s as a viable compute environment for their bioinformatic pipelines. In this article, Ben Sherman, Seqera Lab’s master of all things Kubernetes, details recent improvements to the Nextflow/Kubernetes integration.
+image=img/kubernetes-enhancements.jpg
+tags=nextflow, kubernetes
+status=published
+author=Ben Sherman
+icon=ben.jpg
+~~~~~~
+
+Hi, my name is Ben, and I’m a software engineer at Seqera Labs. I joined Seqera in November 2021 after finishing my Ph.D. at Clemson University. I work on a number of things at Seqera, but my primary role is that of a Nextflow core contributor.
```
Comment (ewels) [inline content/blog/2023/nextflow-kubernetes-enhancements.md:12]: I find the term "Nextflow core" quite jarring 😅  Whilst accurate, is there another term we can use?

Comment (pditommaso) [reply content/blog/2023/nextflow-kubernetes-enhancements.md:12]: lol, Nextflow core is not nf-core 🤣

Comment (ewels) [reply content/blog/2023/nextflow-kubernetes-enhancements.md:12]: I know! But people call nf-core "Nextflow core" a lot, so I've kind of got used to reading it as such 😅

Comment (pditommaso) [reply content/blog/2023/nextflow-kubernetes-enhancements.md:12]: any suggestion?

Comment (bentsherman) [reply content/blog/2023/nextflow-kubernetes-enhancements.md:12]: Nextflow maintainer?

Comment (ewels) [reply content/blog/2023/nextflow-kubernetes-enhancements.md:12]: Yeah I usually say Nextflow maintainer, or developer 👍🏻

Comment (bentsherman) [reply content/blog/2023/nextflow-kubernetes-enhancements.md:12]:
```suggestion
Hi, my name is Ben, and I’m a software engineer at Seqera Labs. I joined Seqera in November 2021 after finishing my Ph.D. at Clemson University. I work on a number of things at Seqera, but my primary role is that of a Nextflow maintainer.
```

File: content/blog/2023/nextflow-kubernetes-enhancements.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
+title=Nextflow Kubernetes enhancements
```
Comment (bentsherman) [inline content/blog/2023/nextflow-kubernetes-enhancements.md:1]:
```suggestion
title=The State of Nextflow and Kubernetes
```

**Comment by bentsherman** [comment]
@GJSissons can you update the image to reflect the new title?

**Comment by GJSissons** [comment]
Hi @bentsherman - Yes - I'm updating the graphic tile now to the new title "The State of Nextflow and Kubernetes" thx.

**Comment by pditommaso** [comment]
There are broken links 

Screenshot 2023-03-10 at 12 34 03

**Pull request merged**